easybuild/easyconfigs/y/yaff/yaff-1.6.0-foss-2021b.eb

```diff
@@ -0,0 +1,59 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+#
+# Building this in interactive Slurm session will result in freeze during either
+# runtest phase or sanity_check_commands phase (python -c 'import yaff')
+#
+# If you submit this as non-interactive Slurm job, it will build just fine.
+# Possibly root cause: https://github.com/h5py/h5py/issues/917
+#
+# Updated to foss-2021b. This requires HDF5 to be 1.10.x as newer versions seem
+# to have a different API. Thanks to Simon to point that out.
+# Author: J. Saßmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'yaff'
+version = '1.6.0'
+
+homepage = 'https://molmod.github.io/yaff/'
+description = """Yaff stands for 'Yet another force field'. It is a pythonic force-field code."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+builddependencies = [('pkgconfig', '1.5.5', '-python')]
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('HDF5', '1.10.8'),  # versions 1.12.x does not seem to work
```

Comment:
This requires an exception in the test suite, as the default HDF5 version for `foss/2021b` is 1.12.1:
https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/test/easyconfigs/easyconfigs.py#L474

Reply:
Yes, that makes sense. 
The reason for that YAFF build was as I wanted to use it for LAMMPS, as originally suggested in PR #14653 which I then used in my LAMMPS PR #15869 (update 3). As a newer PR for LAMMPS have been merged, I am not quite sure what to do with it. On one hand I think that test would be good to have, on the other hand there is the problem with the `HDF5` version for the `YAFF` build. 
@boegel Maybe something to touch on at the next EB user meeting?

Reply:
Sorry, that was on my to-do list which is too long. 
I have attached the log-file. The main problem I have is that I don't know how to fix that with the usual way of doing it:
https://portal.hdfgroup.org/display/HDF5/Migrating+from+HDF5+1.10+to+HDF5+1.12
I tried that but I failed miserably. I guess it shows my lack of deeper understanding here. 
This is the log file where I replaced the `1.10.x` version with `1.12.y` but left everything else the same as it is in this PR:

[easybuild-yaff-1.6.0-20220906.101056.LUfUV.log](https://github.com/easybuilders/easybuild-easyconfigs/files/9495216/easybuild-yaff-1.6.0-20220906.101056.LUfUV.log)